Microsoft.Toolkit.Uwp/Helpers/ObjectStorage/IObjectStorageHelper.cs

```diff
@@ -45,6 +64,16 @@ public interface IObjectStorageHelper
         void Save<T>(string key, T value);
 
         /// <summary>
+        /// Save single item by its key in a composite.
```

Comment:
Can you detail in the XML docs why we should use composite save instead of the simple one ?